core/genesis.go

```diff
@@ -57,6 +57,7 @@ func WriteGenesisBlock(chainDb ethdb.Database, reader io.Reader) (*types.Block,
 			Code    string
 			Storage map[string]string
 			Balance string
+			Nonce   string
```

Comment:
Not sure if applicable, but perhaps we could also change the type here to `hexutils.BigInt` or something (not sure the syntax), which could ensure correct parsing without manually having to call common.String2Something. @fjl any input? You are more familiar with the hexutils encoding stuff.